pkg/bundle/kube_version.go

```diff
@@ -0,0 +1,37 @@
+package bundle
+
+import (
+	_ "embed"
+	"fmt"
+	"strings"
+
+	"k8s.io/client-go/discovery"
+)
+
+type KubeVersionClient interface {
+	// ApiVersion returns the Kubernetes API version
+	ApiVersion() (string, error)
+}
+
+type kubeVersionClient struct {
+	kubeServerVersion discovery.ServerVersionInterface
+}
+
+func NewKubeVersionClient(serverVersion discovery.ServerVersionInterface) (manager *kubeVersionClient) {
```

Comment:
```suggestion
func NewKubeVersionClient(serverVersion discovery.ServerVersionInterface) (*kubeVersionClient) {
```

Nitpick really, but naming return values is best avoided, unless you're documenting what they are via names. The names get stale, or allow naked returns, which are best avoided.